cli/azd/pkg/tools/azcli/webapp.go

```diff
@@ -49,19 +49,20 @@ func (cli *azCli) appService(
 	return &webApp, nil
 }
 
+func isLinuxWebApp(response *armappservice.WebAppsClientGetResponse) bool {
+	if response.Properties != nil && response.Properties.SiteConfig != nil &&
+		response.Properties.SiteConfig.LinuxFxVersion != nil {
+		return true
+	}
+	return false
+}
+
 func (cli *azCli) appServiceRepositoryHost(
```

Comment:
I think removing reference here to `cli` works? Just `func appServiceRepositoryHost(..)` would do